new/LIT6952Tasqwaqu.xml

```diff
@@ -0,0 +1,83 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6952Tasqwaqu" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ታስቈቁ፡ ጥቀ፡ ሰቃቀወ፡ ልብ፡ ወተናዝዞ፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Tāsqʷaqu ṭǝqqa saqāqaw lǝbb watanāzǝzo...  (Qǝne of mawaddǝs type)</title>
+                <title xml:lang="en" corresp="#t1">She is crying exceedingly, lamentation of heart and consolation...</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DN"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p>This poem is referred to as qǝne of the type <foreign xml:lang="gez">mawaddǝs</foreign>,
+                    in <ref type="mss" corresp="ESdd037"/>, and attributed to <persName ref="PRS14409Yohannes"/>.</p>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
```

Comment:
In case the information is available, an abstract as in the preceding record would be nice (here and below)

Reply:
ok